pytest.ini

```diff
@@ -0,0 +1,4 @@
+[pytest]
```

Comment:
we already have a pytest section in setup.cfg

Reply:
Got it.  Will move to there.